code/modules/admin/verbs/adminhelp.dm

```diff
@@ -140,7 +140,7 @@
 		if(!adm["afk"] && !adm["stealth"] && !adm["noflags"])
 			send2irc(source, "[msg] - No admins online")
 		else
-			send2irc(source, "[msg] - All admins AFK ([adm["afk"]]/[adm["total"]]), stealthminned ([adm["stealth"]]/[adm["total"]]), or lack [requiredflags] ([adm["noflags"]]/[adm["total"]])")
+			send2irc(source, "[msg] - All admins AFK ([adm["afk"]]/[adm["total"]]), stealthminned ([adm["stealth"]]/[adm["total"]]), or have no power ([adm["noflags"]]/[adm["total"]])")

```

Comment:
It's not always ban flags since it's an argument but fair enough. Is there some proc that can turn the bitflag into a human-readable string?